Customer.java

```diff
@@ -15,18 +15,70 @@
     private int ID;
     private String deliveryAddressLine2;
     private String deliveryPostalCode;
+    private String customerName;
+    private String customerCity;
+    private int longitude;
+    private int latitude;
     
     public Customer(int ID) {
-        
+        //c.CustomerName, c.longitude, c.latitude, c.DeliveryAddressLine2, c.DeliveryPostalCode FROM c.customers
```

Comment:
moet dit niet weg beetje onduidelijk. BV:
```suggestion
        //initializing customer from database
```